libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextSelectors.ts

```diff
@@ -63,16 +78,22 @@ export const selectFilterContextDefinition = createSelector(selectSelf, (filterC
 /**
  * Selects dashboard's filter context identity.
  *
+ * @remarks
  * The identity may be undefined in two circumstances:
  *
  * -  a new, yet unsaved dashboard; the filter context is saved together with the dashboard and after the
  *    save the identity will be known and added
+ *
  * -  export of an existing, saved dashboard; during the export the dashboard receives a temporary
  *    filter context that represents values of filters at the time the export was initiated - which may
  *    be different from what is saved in the filter context itself. that temporary context is not
  *    persistent and lives only for that particular export operation.
  *
- * @alpha
+ * Invocations before initialization lead to invariant errors.
+ *
+ * @returns a {@link @gooddata/sdk-backend-spi#IDashboardObjectIdentity} or undefined, if the filter context identity is not set.
+ *
+ * @public
```

Comment:
I wonder if this selector would be useful to someone other than us :-/ maybe we should keep it internal?